airflow/contrib/hooks/emr_hook.py

```diff
@@ -36,6 +36,26 @@ def get_conn(self):
         self.conn = self.get_client_type('emr', self.region_name)
```

Comment:
```suggestion
        if not self.conn:
            self.conn = self.get_client_type('emr', self.region_name)
```

(and do `self.conn = None` inside `__init__`)